internal/mode/static/state/graph/httproute.go

```diff
@@ -423,6 +424,61 @@ func validateFilterHeaderModifierFields(
 	return allErrs
 }
 
+func validateFilterResponseHeaderModifier(
+	validator validation.HTTPFieldsValidator,
+	responseHeaderModifier *v1.HTTPHeaderFilter,
+	filterPath *field.Path,
+) field.ErrorList {
+	if errList := validateFilterHeaderModifier(validator, responseHeaderModifier, filterPath); errList != nil {
+		return errList
+	}
+	var allErrs field.ErrorList
+	disallowedResponseHeaderSet := map[string]struct{}{
```

Comment:
How did you come up with this set, are there any resources that could be linked to? Also, any thoughts on pulling this out and putting it as a constant outside of the function so it could be easier to modify and find?

Reply:
I am still [working](https://github.com/nginxinc/nginx-gateway-fabric/pull/1880#discussion_r1586920876) on simplifying this function. These are some discussions on the old [PR](https://github.com/nginxinc/nginx-gateway-fabric/pull/1494#pullrequestreview-1891609353)  and why these fields are disallowed. Please refer to this and let me know if you have any more questions.